services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss

```diff
@@ -102,9 +102,16 @@
 .content {
   width: calc(100vw - var(--scrollbar-width));
   max-width: none;
+  // height of sticky header + search = 96px
+  height: calc(100vh - convert.rem(96px));
```

Comment:
the last section get a little bit cut off for me when scrolled all the way, maybe we should reduce the height a smaller amount? 
<img width="408" alt="image" src="https://user-images.githubusercontent.com/40550942/186942037-995b5045-46b9-45d8-894f-461946842a71.png">

Reply:
Ah ok, I think the issue is the extra space above the search when you open the filter. Looking into this.